WeakAurasOptions/Cache.lua

```diff
@@ -129,15 +129,22 @@ function spellCache.BestKeyMatch(nearkey)
   local bestKey = "";
   local bestDistance = math.huge;
   local partialMatches = {};
-  for key, value in pairs(cache) do
+  local lowerMatch
+  for key in pairs(cache) do
+    if key == newkey then
+      return key
+    end
```

Comment:
That iteration is not needed, after all. This can directly check cache[nearkey].